### ISSUE TITLE
Conversational AI Twilio + Python

### ISSUE THREAD

**Issue by Bayesian4042** [elevenlabs/elevenlabs-docs#519]
Path: /conversational-ai/guides/conversational-ai-twilio
The code given in the Path,  not able to handle the multiple calls as previous call is not properly cleaned up and next call just throw errors:
sent 1000 (OK); no close frame received

**Comment by louisjoecodes** [comment]
Hey - thank you for raising this to us. We will act on this today - will keep this issue up to date.

**Comment by dhavalmehtas** [comment]
Also, it doesn't end well—after ending the call and calling again, it gives an error message that "WebSocket closed, stopping audio processing," and you have to re-run the application.

**Comment by louisjoecodes** [comment]
Roger that - I am working on updating the code.

**Comment by dhavalmehtas** [comment]
not yet, It seems it has to do something with how conversation.end_session() works. I tried different approaches but I guess the fix has to be in conversation.py. still checking..

**Comment by Bayesian4042** [comment]
@louisjoecodes I also get this error, even though the call has been disconnected:
websockets.exceptions.ConnectionClosedError: received 1011 (internal error) Unexpected error in websocket endpoint: Conversation has exceeded maximum duration; then sent 1011 (internal error) Unexpected error in websocket endpoint: Conversation has exceeded maximum duration

**Comment by louisjoecodes** [comment]
Hey folks - while we recreate the guide for Python - I have just pushed a full example project using Javascript:
https://github.com/elevenlabs/elevenlabs-examples/tree/main/examples/conversational-ai/twilio/javascript

Thank you for your patience!

**Comment by sh3pst1** [comment]
> Hey folks - while we recreate the guide for Python - I have just pushed a full example project using Javascript: https://github.com/elevenlabs/elevenlabs-examples/tree/main/examples/conversational-ai/twilio/javascript
> 
> Thank you for your patience!

Hey, this one worked fine until a couple of hours ago. Then it all stopped working. 
The "First message" get cracked up, none of my input seems to go in and the call disconnects.
Same result running local as in production.
Any ideas ?

**Comment by louisjoecodes** [comment]
Hey @sh3pst1 sorry you are experiencing this - could you confirm which SDK/version you are using?

I have updated our Swift SDK as we now require an initiation web socket message to be sent from the client to begin the conversation.
```
{"type":"conversation_initiation_client_data"}
```

**Comment by sh3pst1** [comment]
> Hey @sh3pst1 sorry you are experiencing this - could you confirm which SDK/version you are using?
> 
> I have updated our Swift SDK as we now require an initiation web socket message to be sent from the client to begin the conversation.
> 
> ```
> {"type":"conversation_initiation_client_data"}
> ```

Yes, I followed your guide from here https://github.com/elevenlabs/elevenlabs-examples/tree/main/examples/conversational-ai/twilio/javascript

**Comment by sh3pst1** [comment]
> [@sh3pst1](https://github.com/sh3pst1) are you still having issues? If so could you provide some logs or error messages so we can narrow down what the issue is?

Nope, this has not been a problem for some time now.
